packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -1688,13 +1710,12 @@ class ColorScheme with Diagnosticable {
     final List<int> scoredResults = Score.score(colorToCount, desired: 1);
     final ui.Color baseColor = Color(scoredResults.first);
 
-    final SchemeTonalSpot scheme;
-    switch (brightness) {
-      case Brightness.light:
-        scheme = SchemeTonalSpot(sourceColorHct: Hct.fromInt(baseColor.value), isDark: false, contrastLevel: 0.0);
-      case Brightness.dark:
-        scheme = SchemeTonalSpot(sourceColorHct: Hct.fromInt(baseColor.value), isDark: true, contrastLevel: 0.0);
-    }
+    final bool isDark = brightness == Brightness.dark;
```

Comment:
Same suggestion as earlier. You might want to factor this whole thing out, to ensure that the two copies of the code never diverge.